.github/scripts/end2end/run-e2e-ctst.sh

```diff
@@ -36,4 +53,5 @@ kubectl run $POD_NAME \
         --attach=True \
         --image-pull-policy=Always \
         --env=TARGET_VERSION=$VERSION  \
-        -- ./run "$COMMAND" $WORLD_PARAMETERS "--parallel $PARALLEL_RUNS --retry $RETRIES --retry-tag-filter @Flaky"
+        --env=VERBOSE=1 \
```

Comment:
Note for other reviewers: I think we should keep this flag set in the CI, t ease debugging in case of error, as this flag will display all the aws cli commands issued to Zenko, with all details about the responses.